broke-protocol/src/main/java/org/dsngroup/broke/protocol/ConnectMessageHeader.java

```diff
@@ -0,0 +1,24 @@
+package org.dsngroup.broke.protocol;
+
+/**
+ * The header class of the CONNECT message
+ * The class has an addition topic attribute than the basic message.
+ * */
+public class ConnectMessageHeader extends MessageHeader {
+
+    /**
+     * The constructor, construct the message header by a raw header string
+     * @param rawHeader raw header string
+     * @throws RuntimeException Wrong option fields.
+     * */
+    public ConnectMessageHeader(String rawHeader) throws Exception {
+        String[] fields = rawHeader.split(",");
+        for (int i = 0; i < fields.length; i++) {
+            String[] optionSplit = fields[i].split(":", 2);
+            switch (optionSplit[0].toUpperCase()) {
+                default:
+                    setOptions(optionSplit);
+            }
```

Comment:
Looks weird?

Reply:
Yes, because this message doesn't contain additional options. The switch statement can be removed.

Reply:
But there may be more options in the future, maybe I'll leave this weird code?

Reply:
ok, add a TODO comment